Code/GraphMol/FindStereo.cpp

```diff
@@ -880,15 +891,43 @@ void cleanMolStereo(ROMol &mol, const boost::dynamic_bitset<> &fixedAtoms,
       }
     }
   }
+
+  bool removedStereo = false;
   for (auto i = 0u; i < mol.getNumBonds(); ++i) {
     auto bond = mol.getBondWithIdx(i);
     if (!fixedBonds[i] && knownBonds[i]) {
       bond->setStereo(Bond::BondStereo::STEREONONE);
       bond->setBondDir(Bond::BondDir::NONE);
       bond->getStereoAtoms().clear();
+      removedStereo = true;
+    }
+  }
+
+  // remove any slash bond dirs that do not have a stereo neighbor
+
+  if (removedStereo) {
```

Comment:
Would it be better just to check the neighbors of the bonds you've just removed stereo from?

Reply:
The slashes and backslashed could, and often do have stereo double bonds on both ends.  Just because the stereo indication on one end was removed does not mean that the slash or backslash should be removed.  So it is done after all the stereo indicattions on double bonds and done, then we check to see if any slashes/backslashes had NO stereo bonds on them